internal/driver/glfw/window.go

```diff
@@ -536,7 +536,20 @@ func (w *window) processMouseClicked(button desktop.MouseButton, action action,
 	}
 
 	if wid, ok := co.(fyne.Focusable); !ok || wid != w.canvas.Focused() {
-		w.canvas.Unfocus()
+		ignore := false
```

Comment:
Reading the code here I can't really understand what effect this change is having and why it was necessary, but could it possibly be impacting those with custom lists that handle selection/focus themselves?

Reply:
Previously if you tapped an item that was not Focusable inside/over a focused item it would remove the focus (assuming they were different widgets).
What this does it changes it so that whilst "over" a focused widget it will only change focus if the child is Focusable.

Reply:
So for those with Focusable child elements who worked around this previous list focus handling it should still work.

Reply:
Sounds like a good change 👍